mission_control/navigator_launch/launch/cameras.launch

```diff
@@ -0,0 +1,80 @@
+<launch>
+  <!-- This is the default pointgrey launch file with our id's and our names  -->
+
+  <arg name="front_camera_serial" default="15223362" />
+  <arg name="front_camera_calibrated" default="0" />
+  <arg name="down_camera_serial" default="10282860" />
+  <arg name="down_camera_calibrated" default="0" />
```

Comment:
There was a camera launch on the boat that would launch just one camera but the image was black and white in rqt. This launch file launches all three cameras with the processing. I just wanted to get it working to bag on Saturday. If you'd like to scrap it afterwards, i won't get in your way